workers/deployment_manager/DeploymentManager/Log.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+using System.Threading;
+using Improbable.Worker;
+
+namespace DeploymentManager
+{
+    public class Log
+    {
+        private static readonly Mutex logMutex = new Mutex();
+
+        private static StreamWriter logStream;
+
+        public static void Print(LogLevel level, string message, Connection connection = null)
+        {
+            logMutex.WaitOne();
```

Comment:
Do we need a full mutex or can we just `lock` on the StreamWriter object or similar?

Reply:
probably a simple `lock` would work. gonna try